### DISCUSSION TITLE
[Crash] - connected or edition mode

### DISCUSSION THREAD

**Discussion by watchix** [cedricp/ddt4all#867] [Q&A]
Hello,

I'm having a crash after double-clicking on every object in this left bar :
![image](https://github.com/cedricp/ddt4all/assets/2278299/335d16b4-7467-469e-a83c-4d27b7790342)

Logs of the crash :
![ddt4all_crash](https://github.com/cedricp/ddt4all/assets/2278299/b151f3de-eddc-442e-8136-9ff8fdb2d23f)

Regards,

**Comment by Furtif** [comment]
it seems not to be a ddt4all error but the ecu in question

I will convert to discussion.

**Comment by drder66** [comment]
This doesn't seem to be about a single ECU. I've had three separate, differently dated copies of the database, and every ECU (Ive tried 10 completely different ones from different vehicles) I try fails the same way. 

It would be helpful to know what DDT2000 database(s) have been validated as working with this tool

**Comment by drder66** [comment]
You might try the answer I posed to myself over on #864. Your error messages are exactly the same as mine were before applying the fix.
